packages/js/settings-editor/package.json

```diff
@@ -34,6 +34,7 @@
 		"access": "public"
 	},
 	"dependencies": {
+		"@automattic/site-admin": "^0.0.1",
```

Comment:
It looks like a new dependency, we should add it to `.syncpackrc` to ensure the monorepo will use same version of this dependency.

Reply:
Thanks!